my-app/src/app/shared/index.ts

```diff
@@ -1,5 +1,2 @@
-export * from './pipes/duration-handler/duration-handler.pipe';
-export * from './pipes/filter/filter.pipe';
-export * from './pipes/orderBy/order-by.pipe';
-export * from './directives/is-course-fresh.directive';
-export * from './types';
+export * from '../modules/courses-page/core/models/course.model';
```

Comment:
Should it be in `shared` too?

Reply:
In last conversation of this MR we founded out that yes)